test/Engine/ProtoTest/MultiLangTests/NullInputTest.cs

```diff
@@ -0,0 +1,26 @@
+using NUnit.Framework;
+using ProtoCore.DSASM.Mirror;
+namespace ProtoTest.MultiLangTests
+{
+    class NullInputTest : ProtoTestBase
+    {
+        public override void Setup()
+        {
+            base.Setup();
+
+            // TODO: Determine if this is necessary for this test
+            FFITarget.DisposeCounter.Reset(0);
+        }
+
+        [Test]
+        public void DYN_2093_NullInput_Flatten()
+        {
+            var code =
+@"
+import(""DSCoreNodes.dll"");
+DSCore.List.flatten(null);
+  ";
+            thisTest.RunScriptSource(code);
+        }
+    }
+}
```

Comment:
Please also give a non-null input list to the `List.flatten` method and assert that it returns null and does not crash.